### PR TITLE
oasisriddle: Translate instructions to English

### PR DESCRIPTION
**Pull request by matejanez**
English version of the riddle text

### CODE REVIEW

File: oasisriddle/docs/riddle_eng [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Oasis Riddle
+
+Developers at [Oasis Network][oasis-network] have created a short riddle
+that unlocks the path to a ROSE treasure. They hid it on the [Oasis Sapphire][oasis-explorer]
+blockchain and your job now is to write a script connecting to the
+smart contracts already deployed on Sapphire that will enable you to obtain
+the secret questions. If you manage to answer all the questions correctly,
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:7]: It's just one question, but random based on who is asking ;)

File: oasisriddle/docs/riddle_eng [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Oasis Riddle
+
+Developers at [Oasis Network][oasis-network] have created a short riddle
+that unlocks the path to a ROSE treasure. They hid it on the [Oasis Sapphire][oasis-explorer]
+blockchain and your job now is to write a script connecting to the
+smart contracts already deployed on Sapphire that will enable you to obtain
+the secret questions. If you manage to answer all the questions correctly, 
+you'll be able to collect your ROSE treasure! 🌹
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:10]: This title was for a national programming competition. For Consensus, I suggest something else:

```suggestion
## I like blockchains, but I'm not a skilled programmer and  I haven't really written any smart contract yet
```

File: oasisriddle/docs/riddle_eng [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+If you know C, Python or Javascript, you have all the basics covered,
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:12]:
```suggestion
Don't worry! If you have some programming basics in Javascript, Python, C, or other programming languages, you have all the basics covered. You'll quickly learn everything else on the go 🎓
```

File: oasisriddle/docs/riddle_eng [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+that unlocks the path to a ROSE treasure. They hid it on the [Oasis Sapphire][oasis-explorer]
+blockchain and your job now is to write a script connecting to the
+smart contracts already deployed on Sapphire that will enable you to obtain
+the secret questions. If you manage to answer all the questions correctly, 
+you'll be able to collect your ROSE treasure! 🌹
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
+
+If you know C, Python or Javascript, you have all the basics covered, 
+everything else you'll be able to learn during the challange. 🎓
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:13]:
```suggestion
```

File: oasisriddle/docs/riddle_eng [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Blockchains are generally not a simple thing and developing dApps has lot
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:15]:
```suggestion
Generally, blockchains are not easy to understand and developing dApps has
```

File: oasisriddle/docs/riddle_eng [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+the secret questions. If you manage to answer all the questions correctly, 
+you'll be able to collect your ROSE treasure! 🌹
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
+
+If you know C, Python or Javascript, you have all the basics covered, 
+everything else you'll be able to learn during the challange. 🎓
+
+Blockchains are generally not a simple thing and developing dApps has lot 
+more limitations compared to desktop or web apps. That is why one needs
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:16]:
```suggestion
some limitations compared to desktop or web apps. That is why one needs 
```

File: oasisriddle/docs/riddle_eng [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+you'll be able to collect your ROSE treasure! 🌹
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
+
+If you know C, Python or Javascript, you have all the basics covered, 
+everything else you'll be able to learn during the challange. 🎓
+
+Blockchains are generally not a simple thing and developing dApps has lot 
+more limitations compared to desktop or web apps. That is why one needs 
+an open mind while also having great understanding of algorhythms,
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:17]:
```suggestion
an open mind while also having understanding of the algorithms, 
```

File: oasisriddle/docs/riddle_eng [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+## What if I don't exactly know how Blockchains, Ethereum, Solidity, Dapps work?
+
+If you know C, Python or Javascript, you have all the basics covered, 
+everything else you'll be able to learn during the challange. 🎓
+
+Blockchains are generally not a simple thing and developing dApps has lot 
+more limitations compared to desktop or web apps. That is why one needs 
+an open mind while also having great understanding of algorhythms, 
+data structures, writting effecient code, bug hunting skills, thorough
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:18]:
```suggestion
data structures, writting efficient code, bug hunting skills, thorough
```

File: oasisriddle/docs/riddle_eng [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+understanding of the programs from a cryptographis and security point of view, etc. 🤓
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:19]:
```suggestion
understanding of the programs from a cryptographic and security point of view, etc. 🤓
```

File: oasisriddle/docs/riddle_eng [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+## OK. What do I need to do?
+
+1. First join our `#sapphire-paratime` channel on our official
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:23]:
```suggestion
1. First, join our `#sapphire-paratime` channel on our official
```

File: oasisriddle/docs/riddle_eng [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+   Even if you don't need help, stop by and say hi, everybody will be happy to meet you. 😉
+
+2. If this is your first time developing a dApp, you can watch a 
+   [short video on Ethereum smart contracts][ethereum-tutorial]. 💡
+
+3. When you're ready to move further, check out our 
+   [quickstart guide to Sapphire confidential smart contracs][oasis-sapphire-quickstart]. 💎
+
+4. After you're done with both tutorials, it's time to solve the riddle. You can find it at
+   `0x4874c1EC02c122Dfb29C994264dFCC4aD9FA8b96` on [Oasis Sapphire Testnet][oasis-explorer-testnet].
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:34]:
```suggestion
   `0xTODO` on [Oasis Sapphire Testnet][oasis-explorer-testnet]. 
```

File: oasisriddle/docs/riddle_eng [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+   
+   The ABI of the contract is:
+
+```solidity
+function getQuestion(string memory coupon) external view returns (string memory)
+function submitAnswer(string memory coupon, string memory answer) external
+function claimReward(string memory coupon) external view returns (string memory)
+```
+
+5. Your task is to write a script with the help of the newly obtained skills and tolls,
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:44]:
```suggestion
5. Your task is to write a script with the help of the newly obtained skills and tools, 
```

File: oasisriddle/docs/riddle_eng [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+   The ABI of the contract is:
+
+```solidity
+function getQuestion(string memory coupon) external view returns (string memory)
+function submitAnswer(string memory coupon, string memory answer) external
+function claimReward(string memory coupon) external view returns (string memory)
+```
+
+5. Your task is to write a script with the help of the newly obtained skills and tolls, 
+   that communicates with the riddle's smart contract. First use a confidential call
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:45]:
```suggestion
   that communicates with the riddle's smart contract. First, use a confidential call
```

File: oasisriddle/docs/riddle_eng [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+   `getQuestion()` to get the secret questions tied to your unique ID. Then create
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:46]:
```suggestion
   `getQuestion()` to get the secret questions tied to your unique ID. Then, create
```

File: oasisriddle/docs/riddle_eng [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+   a secret answer and send it back with a confidential transaciton via 
+   `submitAnswer()`. If the answer is correct, you'll be able to send a confidential 
+   `claimReward()` call in the next block to obtain the private key 
+   to your ROSE treasure on [Oasis Sapphire Mainnet][oasis-explorer]. 🎉
+
+6. Congratulations, you've unlocked your ROSE treasure! You can now use your ROSE coins
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:52]:
```suggestion
6. Congratulations, you've unlocked your ROSE treasure! You can now use your ROSE
```

File: oasisriddle/docs/riddle_eng [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+   that communicates with the riddle's smart contract. First use a confidential call
+   `getQuestion()` to get the secret questions tied to your unique ID. Then create
+   a secret answer and send it back with a confidential transaciton via 
+   `submitAnswer()`. If the answer is correct, you'll be able to send a confidential 
+   `claimReward()` call in the next block to obtain the private key 
+   to your ROSE treasure on [Oasis Sapphire Mainnet][oasis-explorer]. 🎉
+
+6. Congratulations, you've unlocked your ROSE treasure! You can now use your ROSE coins 
+   to create new dApps on Oasis and be among the first to utilize confidential smart contracts. 
+   Or you can try to cash them in for a beer with your friends! 🍻
```
Comment (matevz) [inline oasisriddle/docs/riddle_eng:54]: ?

```suggestion
   Or you can try to cash them in for a [beer with your friends][bitcoin-map]! 🍻
```

**Review by matevz — CHANGES_REQUESTED** [review]

File: oasisriddle/docs/riddle_eng (hunk 16 above)
Comment (matejanez) [reply oasisriddle/docs/riddle_eng:53]: The easiest way to cash them out is through an exchange but don't want to promote that, going from ROSE to bitcoin, you probably don't get anything out :D That's why I wrote just try to cash them out for a beer without any suggestions

**Review by matejanez — COMMENTED** [review]
Reviewed and everything resolved, except for the beer one, left you a comment. Oh and it didn't allow me to name it riddle.md as it already exists and couldn't overwrite it

**Review by matejanez — COMMENTED** [review]
Looks good to me

**Comment by matevz** [comment]
Can you rename riddle_eng.md to riddle.md and merge the PR?

**Review by matevz — APPROVED** [review]

**Comment by matevz** [comment]
Superseeded by #1. Thanks. Closing.

**Pull request closed without merging**